pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -1042,3 +1152,31 @@ func max(x, y int) int {
 	}
 	return y
 }
+
+func (c *InstallerController) startupMonitorEnabled() (bool, error) {
```

Comment:
we are going to need something similar in `kas-o` to know when to install the startup monitor.
wondering if this could be moved into `kas-o`?

Reply:
this is just the reader. Just do the same in ckao. It's temporary and will go away eventually.

Reply:
actually it is replaced by the infrastructure resource test. So we have to expose it.

Reply:
See last commit. Not nice, but I don't see a better way.